HiP-DataStore/Startup.cs

```diff
@@ -91,11 +93,13 @@ public void ConfigureServices(IServiceCollection services)
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory,
-            IOptions<CorsConfig> corsConfig)
-        {
-            loggerFactory.AddConsole(Configuration.GetSection("Logging"))
-                         .AddDebug();
+            IOptions<CorsConfig> corsConfig,IOptions<LoggingConfig> loggingConfig)
+            {
+            loggerFactory.AddConsole (Configuration.GetSection ("Logging"))
+                .AddDebug ()
+                .AddHipLogger (loggingConfig.Value);
 
+            loggerFactory.CreateLogger("ApplicationStartup").LogInformation("HiPDataStore Configured Successfully");
```

Comment:
Since this is done every time at startup, I think the message is misleading. It would be better if the logs says something like "DataStore started successfully".

Reply:
I have fixed this also change the position of startup log message.